spec/classes/puppet_agent_spec.rb

```diff
@@ -94,7 +94,7 @@
 
         it { is_expected.to contain_class('puppet::agent::service::systemd').with_enabled(false) }
         case os
-        when /\A(debian|redhat|centos|scientific|fedora|ubuntu|sles|archlinux)-/
+        when /\A(debian|redhat|centos|scientific|fedora|ubuntu|sles|archlinux|oraclelinux|almalinux|rocky)-/
```

Comment:
you can argue this should rather match on family, not the os itself, but I am lazy